scripts/build/Jenkins/Jenkinsfile

```
@@ -1192,7 +1192,7 @@ try {
                     timestamps {
                         repositoryUrl = scm.getUserRemoteConfigs()[0].getUrl()
                         // repositoryName is the full repository name
-                        repositoryName = (repositoryUrl =~ /https:\/\/github.com\/(.*)\.git/)[0][1]
+                        repositoryName = (repositoryUrl =~ /https:\/\/(.*)\/(.*)\.git/)[0][1]
```

Should `.+` be used here instead to make sure there is at least one character between https:// and `/`.

```suggestion
                        repositoryName = (repositoryUrl =~ /https:\/\/(.+)\/(.+)\.git/)[0][1]
```

---

Yup, that's fine. It'll still fail out if the URL is invalid, but better to be sure nothing weird is injected